src/pages/player-info.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Information For Players
+subtitle: >-
+image: 
+template: page
+---
+# Overview
+For those who aren’t familiar, [Jackbox Party Games](https://www.jackboxgames.com/) are a set of quirky, well, party games. They consist of fun takes on Pictionary, Balderdash, Trivial Pursuit, and more. In the world of isolation, Jackbox games have been used via Zoom and other platforms to play with friends and family members around the world.
+
+DevOps Party Games takes this idea and tilts it on its head by adding DevOps-inspired content to the Jackbox games [Drawful 2](https://www.jackboxgames.com/drawful-two/) and [Quiplash 2](https://www.jackboxgames.com/quiplash-two-interlashional/), and then streams it live via [Twitch](https://twitch.tv/mattstratton) for a worldwide audience to watch, comment, and hopefully be entertained. In addition, the hosts ([Matt Stratton](https://twitter.com/mattstratton), [Jeremy Meiss](https://twitter.com/IAmJerdog), and [Dan Maher](https://twitter.com/phrawzty)) will provide color commentary, much like a modern day Cotton McKnight and Pepper Brooks (announcers from Dodgeball). The goal here is to have fun and not take things too seriously.
+
+If you’d like to see an example, previous episodes can be viewed at [http://matty.wtf/devops-party-games](http://matty.wtf/devops-party-games).
+
+# The Format
+There will be two rounds - one round of Quiplash, and one round of Drawful.
```

Comment:
```suggestion
There are three rounds: one each of Quiplash, Thriplash, and Drawful.
```

Reply:
hmm. technically thriplash (from the player perspective) is just part of the quiplash game. not sure if this will add confusion, but I also have no strong objection.

Reply:
the reason that I don't say you separate them is that they are scheduled as one thing, and also they are ranked together for TLB